### PR TITLE
Promote idOS integration into production

### PR DESCRIPTION
**Pull request by shelegdmitriy**

### CODE REVIEW

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -135,15 +135,21 @@ export default function App({ Component, pageProps }: AppPropsWithLayout) {
...
-          position: 'absolute',
-          top: '50%',
-          left: '50%',
-          width: '150px',
-          height: '30px',
-          margin: 0,
-          padding: 0,
-        }}
+        style={
+          !router.route.startsWith('/settings')
```
Comment (charleslavon) [inline src/pages/_app.tsx:139]: @shelegdmitriy why is this necessary?

Comment (shelegdmitriy) [reply src/pages/_app.tsx:139]: To make sure that we show the idOS button (there is an iframe which renders inside of this container) only on the settings page. If we move that to the `/settings` page then initialization fails and  [methodNames](https://github.com/near/near-discovery/blob/7c70e46955f44f4f1d297e99bfff181b2fff6679/src/components/vm/VmInitializer.tsx#L112) will be undefined

Comment (shelegdmitriy) [reply src/pages/_app.tsx:139]: We need that [methodNames](https://github.com/near/near-discovery/blob/7c70e46955f44f4f1d297e99bfff181b2fff6679/src/components/vm/VmInitializer.tsx#L112) to be added to the wallet selector in other case we can't interact with idOS sdk

**Review by charleslavon — APPROVED** [review]
